app/controllers/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
```

Comment:
combine this to be a one liner and always use single quotes if you are not using interpolation.

```suggestion
      redirect_to @user, notice: 'Welcome to the Sample App!'
```